### ISSUE TITLE
command line arguments??

### ISSUE THREAD

**Issue by Koriebonx98** [MAXBURAOT/SmartSteamEmu#21]
is there away to launch via command line or use as an emulator.
for example, is there away to auto scan folders for "Steam_appid.txt" and then add to launcher with default settings as the game folder had "steam_api.txt" 

im trying to add the program into playnite as an emulator, i want it to scan the "$:\Games\" folder  with the sse launcher as the emulator to achieve this.

**Comment by Nirvloyd** [comment]
> is there away to launch via command line or use as an emulator. for example, is there away to auto scan folders for "Steam_appid.txt" and then add to launcher with default settings as the game folder had "steam_api.txt"
> 
> im trying to add the program into playnite as an emulator, i want it to scan the "$:\Games" folder with the sse launcher as the emulator to achieve this.

Hey,late reply but did you have any luck adding it to playnite?

**Comment by Koriebonx98** [comment]
> Hey,late reply but did you have any luck adding it to playnite?

Yes and no, I made a few addons for playnite. I’ve also started to make my own custom app “Game OS” 

discord: “koriebonx98”
